python/GafferSceneUITest/ParameterInspectorTest.py

```diff
@@ -59,12 +59,12 @@ def testName( self ) :
 		self.assertEqual( inspector.name(), "penumbraAngle" )
 
 	@staticmethod
-	def __inspect( scene, path, parameter, editScope=None, attribute="light" ) :
+	def __inspect( scene, path, parameter, editScope=None, attribute="light", shader = "" ) :
```

Comment:
`parameter` and `shader` are really confusing as separate arguments, especially when they're separated and one is optional. What we're really specifying here is a single `ShaderNetwork.Parameter()` which stores both the shader handle and the parameter handle. I suggest we drop the `shader` parameter and just pass a complete `Parameter`to the `parameter` argument (falling back to the current setup if a string is passed instead).